docs/typescript/composition-api.md

```diff
@@ -271,11 +279,13 @@ interface Book {
 const book: Book = reactive({ title: 'Vue 3 Guide' })
 ```
 
-:::tip
+<box type="info">
```

Comment:
Just curious, why use info here instead of tip?

Reply:
I used tip because the logo for the info box matches the one on the tip in the Vue guide! I could change it to tip as well!

Reply:
Oh I see. Thats fine :) I was just asking haha